src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -883,12 +884,12 @@ javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) description)?
       | SERIAL_FIELD_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) FIELD_NAME)? (WS | NEWLINE)*
           ((WS | NEWLINE) FIELD_TYPE)? (WS | NEWLINE)* ((WS | NEWLINE) description)?
```

Comment:
javadoc tool doesn't complain if no args are supplied to `@serial` `@serialData`  so I didn't touch them. 

`@serialField` behaves strangely as I wasn't able to get it to show on the site rendered by the javadoc tool even after sticking to the syntax and other requisities. **NOTE:** javadoc tool still gives a warning if no args are specified for `@serialField` tag, but since I couldn't get this tag to work properly _(or perhaps it was working properly and my expectations of it were wrong)_, I couldn't figure out what does javadoc tool treat the only argument as by default _i.e [does it treat it as `field-name` or `field-type` or `field-description`](http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#serialField)_ . This needs to be looked into further.

Reply:
> This needs to be looked into further

Please open an issue